server/routes/events.ts

```diff
@@ -41,6 +41,9 @@ events.get('/all', (req: Request, res: Response) => {
       { model: EventParticipant, include: [{ model: User, attributes: ['id', 'name', 'image']}] },
       { model: User, attributes: ['id', 'name', 'image'] },
     ],
+    order: [
+      ['startDate', 'ASC']
```

Comment:
you cant sort events here ... this would break my entire component ... just sort it in react please

Reply:
All this is doing is sorting the events to be in order by date. Stuff like this should be taking place in the backend anyway since doing them in the frontend is more expensive. Is it really gonna break your code though? The way you set up the data previously had all the events occurring on the same exact date.